stories/tree/tree.stories.js

```diff
@@ -1,897 +1,1019 @@
+                        </div>
+                    </li>
+                    <li role="treeitem" aria-level="3" class="fd-tree__item">
+                        <div tabindex="0" class="fd-tree__item-container">
+                            <div class="fd-tree__content">
+                                <span class="sap-icon--history sap-icon--l fd-tree__icon"></span>
+                                <span class="fd-tree__text">Level 3</span>
                             </div>
+                            <button aria-label="Edit button" class="fd-button fd-button--transparent sap-icon--edit"></button>
+                            <button aria-label="Delete button" class="fd-button fd-button--transparent sap-icon--decline"></button>
                         </div>
+                    </li>
+                </ul>
+            </li>
+            <li role="treeitem" aria-level="2" class="fd-tree__item">
+                <div tabindex="0" class="fd-tree__item-container">
+                    <div class="fd-tree__content">
+                        <span class="sap-icon--competitor sap-icon--l fd-tree__icon"></span>
+                        <span class="fd-tree__text">Level 2</span>
+                    </div>
+                    <button aria-label="Edit button" class="fd-button fd-button--transparent sap-icon--edit"></button>
+                    <button aria-label="Delete button" class="fd-button fd-button--transparent sap-icon--decline"></button>
+                </div>
+            </li>
+        </ul>
+    </li>
+    <li role="treeitem" aria-level="1" class="fd-tree__item">
+        <div tabindex="0" class="fd-tree__item-container">
+            <div class="fd-tree__content">
+                <span class="sap-icon--batch-payments sap-icon--l fd-tree__icon"></span>
+                <span class="fd-tree__text">Level 1</span>
+            </div>
+            <button aria-label="Edit button" class="fd-button fd-button--transparent sap-icon--edit"></button>
+            <button aria-label="Delete button" class="fd-button fd-button--transparent sap-icon--decline"></button>
+        </div>
+    </li>
+    <li role="treeitem" aria-level="1" class="fd-tree__item">
+        <div tabindex="0" class="fd-tree__item-container">
+            <div class="fd-tree__content">
+                <span class="sap-icon--favorite sap-icon--l fd-tree__icon"></span>
+                <span class="fd-tree__text">Level 1</span>
+            </div>
+            <button aria-label="Edit button" class="fd-button fd-button--transparent sap-icon--edit"></button>
+            <button aria-label="Delete button" class="fd-button fd-button--transparent sap-icon--decline"></button>
+        </div>
+    </li>
+</ul>
+`;
+
+treeWithExpandedLevel2AndBorderlessDesign.parameters = {
+    docs: {
+        iframeHeight: 400,
+        storyDescription: `
+When Level 2 is expanded to show Level 3, Level 2 changes to be indented by 1rem, Level 3 is indented by 2.5rem.
```

Comment:
You can say "When Level 2 expands to show Level 3, Level 2 indents by 1rem while Level 3 indents by 2.5rem."